expbar/expbar.lua

```diff
@@ -0,0 +1,90 @@
+local math = require('math')
+local player = require('player')
+local client_data = require('client_data')
+local string = require('string')
+local ui = require('ui')
+local windower = require('windower')
+
+local window_state = {
+    title = 'Exp Bar',
+    style = 'chromeless',
+    x = math.floor(windower.settings.ui_size.width / 2) - 350,
+    y = 1,
+    width = 750,
+    height = 16,
+    color = ui.color.rgb(0,0,0,0)
```

Comment:
Spaces after commas (same in other places in the file, not gonna mention them). And if you wanna do me a personal favor add a trailing comma :)